operate/schema/src/main/java/io/camunda/operate/schema/IndexSchemaValidator.java

```diff
@@ -41,7 +44,9 @@ public class IndexSchemaValidator {
 
   @Autowired SchemaManager schemaManager;
 
-  private Set<String> getAllIndexNamesForIndex(String index) {
+  @Autowired private OperateProperties operateProperties;
```

Comment:
Suggest updating to constructor dependency injection wherever possible rather than autowiring member variables